.github/workflows-source/release-testing.yml

```diff
@@ -41,6 +41,17 @@ anchors:
         --privileged --cgroupns host
         -v /cache:/cache -v /var/sysimage:/var/sysimage -v /var/tmp:/var/tmp
     timeout-minutes: 180 # 3 hours
+  dind-small-setup: &dind-small-setup
+    runs-on:
+      group: zh1
+      labels: dind-large
+    container:
+      <<: *image
+      options: >-
+        -e NODE_NAME
+        --privileged --cgroupns host
+        -v /cache:/cache -v /var/sysimage:/var/sysimage -v /var/tmp:/var/tmp
+    timeout-minutes: 180 # 3 hours
```

Comment:
As [commented below](https://github.com/dfinity/ic/pull/1335/files#r1745785810) I would just use `dind-large-setup` and remove this:
```suggestion
```

Reply:
Alright, I dropped it and reverted to a dind-large